js/pro/cex.js

```diff
@@ -0,0 +1,1229 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const { ExchangeError, ArgumentsRequired } = require ('../base/errors');
+const Precise = require ('../base/Precise');
+const cexRest = require ('../cex');
+const { ArrayCacheBySymbolById, ArrayCacheByTimestamp, ArrayCache } = require ('./base/Cache');
+
+//  ---------------------------------------------------------------------------
+
+module.exports = class cex extends cexRest {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'has': {
+                'ws': true,
+                'watchBalance': true,
+                'watchTicker': true,
+                'watchTickers': true,
+                'watchTrades': true,
+                'watchMyTrades': true,
+                'watchOrders': true,
+                'watchOrderBook': true,
+                'watchOHLCV': true,
+                'watchPosition': undefined,
+            },
+            'urls': {
+                'api': {
+                    'ws': 'wss://ws.cex.io/ws',
+                },
+            },
+            'options': {
+                'orderbook': {},
+            },
+            'streaming': {
+            },
+            'exceptions': {
+            },
+        });
+    }
+
+    requestId () {
+        const requestId = this.sum (this.safeInteger (this.options, 'requestId', 0), 1);
+        this.options['requestId'] = requestId;
+        return requestId;
+    }
+
+    async watchBalance (params = {}) {
+        /**
+         * @method
+         * @name cex#watchBalance
+         * @description query for balance and get the amount of funds available for trading or funds locked in orders
+         * @see https://cex.io/websocket-api#get-balance
+         * @param {object} params extra parameters specific to the cex api endpoint
+         * @returns {object} a [balance structure]{@link https://docs.ccxt.com/en/latest/manual.html?#balance-structure}
+         */
+        await this.authenticate (params);
```

Comment:
Same this is crashing in python for some reason 
![image](https://user-images.githubusercontent.com/43336371/202491560-077c49e8-5db2-44b9-8e14-c5111c35adcb.png)